interfaces/IBF-dashboard/src/app/components/event-speech-bubble/event-speech-bubble.component.ts

```diff
@@ -76,13 +76,31 @@ export class EventSpeechBubbleComponent implements AfterViewChecked, OnDestroy {
 
     if (this.event) {
       this.event.header = this.getHeader(this.event);
+      this.event.mainExposureValueSum = this.getEventMainExposureValue(
+        this.event,
+        this.mainExposureIndicatorNumberFormat,
+      );
     }
   }
 
   ngOnDestroy() {
     this.placeCodeHoverSubscription.unsubscribe();
   }
 
+  private getEventMainExposureValue(
+    event: EventSummary,
+    mainExposureIndicatorNumberFormat: NumberFormat,
+  ) {
+    const sum = event.alertAreas.reduce(
+      (acc, alertArea) => acc + alertArea.mainExposureValue,
+      0,
+    );
+    if (mainExposureIndicatorNumberFormat === NumberFormat.perc) {
```

Comment:
This is still a temporary solution so it needs the comment that we should not be averaging percentages.

Reply:
correct, I will add it.

Reply:
done